custom_components/tuya_local/devices/nince_rgb_led_strip.yaml

```diff
@@ -0,0 +1,158 @@
+name: Nince RGB LED Strip
+products:
+  - id: whomcoylqdl8r4jw
+    name: Smart RGB Strip
+primary_entity:
+  entity: light
+  dps:
+    - id: 20
+      type: boolean
+      name: switch
+    - id: 25
+      type: string
+      name: color_mode
+      mapping:
+        - dps_val: colour
```

Comment:
This does not seem correct. The other values are all hex encoded, but this one is "colour"?

It seems you have moved dp 25 into the place of dp 21 but left this value here. 
I think dp 25 would be better as a separate "Scene" select entity, similar to other devices or as "effect" attribute of light.